### ISSUE TITLE
Flashed session values persist for 1 reload

### ISSUE THREAD

**Issue by faustbrian** [calebporzio/livewire#132]
At the moment flashed session values are persisted for 1 reload after already showing in a component. Below code will show the `success` message after `persist` has been executed but when you refresh the page it will still show, after a second reload it will finally be gone as Laravel has seen that the flashed value was used.

```php
// View
@if(session()->has('success'))
    <div>{{ session()->get('success') }}</div>
@endif

// Component
<?php

namespace App\Http\Livewire;

use Livewire\Component;

class UpdateName extends Component
{
    /** @var string */
    public $name;

    /**
     * Persist all changes.
     */
    public function persist(): void
    {
        // validate...
        // persist...

        session()->flash('success', 'Persisted!');
    }
}
```

This is probably a tricky one to solve due to how Livewire works over an AJAX request so Laravel never saw the session value be used directly over a normal HTTP request.

**Comment by faustbrian** [comment]
https://github.com/faustbrian/livewire/commit/7b7fbe4bc5efa85b8b5c715119508e86a813ca1c

This would be a dirt solution but it solves the issue of the newly flashed messages persisting outside of the Livewire component.

**Comment by calebporzio** [comment]
Merged and tagged: v0.2.5

Thanks!
